tests/queries/0_stateless/01720_type_map_and_casts.sql

```diff
@@ -71,6 +73,9 @@ SELECT m FROM table_map_with_key_integer;
 SELECT m[toInt128(-1)], m[toInt128(0)], m[toInt128(1234567898765432123456789)], m[toInt128(-1234567898765432123456789)] FROM table_map_with_key_integer;
 SELECT m[toInt128(number - 2)] FROM table_map_with_key_integer ARRAY JOIN range(4) AS number;
 
+SELECT m[-1], m[0], m[toInt128(1234567898765432123456789)], m[toInt128(-1234567898765432123456789)] FROM table_map_with_key_integer;
+SELECT m[toUInt64(0)], m[toInt64(0)], m[toUInt8(0)], m[toUInt16(0)] FROM table_map_with_key_integer;
+
```

Comment:
It seems [line 70](
https://github.com/ClickHouse/ClickHouse/blob/68b3cbbfe96e5deb068f3901aa526eb048814478/tests/queries/0_stateless/01720_type_map_and_casts.sql#L70) should be

```
SELECT 'Map(Int128, String)';
```

instead of

```
SELECT 'Map(Int128, Int32)';
```

Could you fix that?